content/en/docs/tasks/security/authentication/authn-policy/snips.sh

```diff
@@ -0,0 +1,506 @@
+#!/bin/bash
```

Comment:
I believe this file shouldn't be included. It might be part of your `master` change as master has a test for this page (https://github.com/istio/istio.io/pull/7396/files#diff-72f5910f0635b96687140aa324274234L9) while `release 1.6` does not.

Reply:
Ahh, thanks for catching.